### DISCUSSION TITLE
How much payouts BTCPayServer can handle at a time?

### DISCUSSION THREAD

**Discussion by ridetothefuture** [btcpayserver/btcpayserver#2611] [Technical Support] [answered]
This is not really a bug but wants to know the power of BTCPayServer. I posted this question on many forums but nobody is responding.

I really like BTCPayServer and wanna use it in a very large scale application where we have 5000 payouts per day. My question is can I select all 5000 payouts together and send them as a single PSBT? Will BTCPayServer be able to handle it? or I will face any issue if I did so?

I'm currently running it on LunaNode small instance

**Comment by Kukks** [comment, marked as answer]
I don't think it would work, there is a maximum bitcoin transaction size that would most likely be reached..  This would be 5000 outputs and a considerable amount of inputs depending on your wallet and amounts
